### ISSUE TITLE
Exception while flushing the report in MAC

### ISSUE THREAD

**Issue by preetipadalia** [extent-framework/extentreports-csharp#94]
Hi,

I am getting following error while trying to flush the report in MAC:
Error Message: Errors while compiling a Template.
  Please try the following to solve the situation:
    * If the problem is about missing/invalid references or multiple defines either try to load 
      the missing references manually (in the compiling appdomain!) or
      Specify your references manually by providing your own IReferenceResolver implementation.
      See https://antaris.github.io/RazorEngine/ReferenceResolver.html for details.
      Currently all references have to be available as files!
    * If you get 'class' does not contain a definition for 'member': 
          try another modelType (for example 'null' to make the model dynamic).
          NOTE: You CANNOT use typeof(dynamic) to make the model dynamic!
      Or try to use static instead of anonymous/dynamic types.
  More details about the error:
  	 - warning: (9, 4) Unnecessary using directive.
  	 - warning: (7, 4) Unnecessary using directive.
  	 - warning: (8, 4) Unnecessary using directive.
  	 - warning: (6, 4) Unnecessary using directive.
  	 - warning: (12, 4) Unnecessary using directive.
  	 - warning: (5, 4) Unnecessary using directive.
  	 - warning: (11, 4) Unnecessary using directive.
  	 - warning: (10, 4) Unnecessary using directive.
  	 - error: (0, 0) Unexpected error writing debug information -- 'COM Interop is not supported on this platform.'

Thanks
Preeti

**Comment by anshooarora** [comment]
@preetipadalia Please share the version of ExtentReports.  Is it https://www.nuget.org/packages/ExtentReports/4.1.0-beta1?

**Comment by preetipadalia** [comment]
@anshooarora  Yes its 4.1.0-Beta1.  I just identified the issue in RazorEngine.NetCore.

**Comment by anshooarora** [comment]
I tried running on MacOS Catalina without issues. I will look into this  

Are you using. NET Core 2.1, or 3.1?

**Comment by preetipadalia** [comment]
I am using .netcoreapp3.1.
